src/js/queryMoviesApi.js

```diff
@@ -0,0 +1,46 @@
+class queryGeneralAPI{ 
+    constructor() {
+        this.page = 1;
+        this.query = '';
+        this.url = ` https://api.themoviedb.org/3/`;
+        this.keyAPI = `b32f977d148061c9ab22a471ff2c7792`; 
+    }
+
+    async generalAPI() {
```

Comment:
I would advice you to pass variables that changes into the method parameters , for example getMoviesApi(page, query = ''), 

So, if you want to retrieve the 5th page with cars you can just invoke getMoviesApi(5, 'car')

Reply:
ok.  i will do it next time

Reply:
it is better to try that this time

Reply:
ok. i will do it next time